service/resource/cloudformation/resource.go

```diff
@@ -110,3 +110,26 @@ func toDeleteStackInput(v interface{}) (awscloudformation.DeleteStackInput, erro
 
 	return deleteStackInput, nil
 }
+
+func getStackOutputValue(outputs []*awscloudformation.Output, key string) string {
+	for _, o := range outputs {
+		if *o.OutputKey == key {
+			return *o.OutputValue
+		}
+	}
+
+	return ""
```

Comment:
IMO this should return an error. We use this function to get a value we expect. If there is no value we should handle that case and get to know about it eventually.

Reply:
Yes agree on this. I think there will be problems reconciling the stack if we can't get all the output values.

Reply:
Sure, done in #502